go/adapter.go

```diff
@@ -83,3 +83,25 @@ func (b *AdapterBase) Stop(wait bool) {
 		b.eventBucket.Wait()
 	}
 }
+
+// Definition of how to filter our Spans to reduce noise
+type SpanFilter struct {
+	MinDuration time.Duration
+}
+
+// Specify options to change what or how the adapter receives ObserveEvents
+type Options struct {
+	SpanFilter        *SpanFilter
+	ChannelBufferSize int
+}
+
+// Create a default configuration
+func NewDefaultOptions() *Options {
+	minDuration, _ := time.ParseDuration("20us")
```

Comment:
can this represented directly as `time.Microsecond * 20`?

https://pkg.go.dev/time#Microsecond